dev-python/imapclient/imapclient-2.0.0.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} pypy )
+
+inherit distutils-r1
+
+MY_PN="IMAPClient"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="easy-to-use, pythonic, and complete IMAP client library"
+HOMEPAGE="http://imapclient.freshfoo.com/"
```

Comment:
This seems to redirect to https://github.com/mjs/imapclient

Reply:
FWICS pypi page also uses the GitHub page as homepage. Also, please append the pypi homepage here as well.